pkg/timing/timing.go

```diff
@@ -42,7 +42,11 @@ func SetTime(timingInfo map[string]string, event string) {
 	if _, set := timingInfo[event]; set {
 		return
 	}
-	timingInfo[event] = strconv.FormatInt(time.Now().UnixNano()/1e6, 10)
+	timingInfo[event] = CurrentTime()
+}
+
+func CurrentTime() string {
+	return strconv.FormatInt(time.Now().UnixNano()/1e6, 10)
```

Comment:
I know this is a rearrangement of what's already here, but it might make sense to do sometime line
```suggestion
	return strconv.Itoa(time.Now().UnixNano()/time.Millisecond)
```